ISISComputingGroup/EPICS-inst_servers
# Ticket3125 run iocs on remote pcs

### Description of work

Adds a remote IOC server.

### To test

https://github.com/ISISComputingGroup/IBEX/issues/3125

Example workflow:
- Follow steps at https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Remote-IOCs#setup to get a basic remote IOC setup working
- Change macros in local config
- Check that macros are correctly applied to remote IOC
- As above for PV values
- Check that access security is set correctly - i.e. `cainfo` returns `read, write` from your machine and only `read` from other machines (note: other machines have to go through the site gateway, this may take multiple minutes to connect)

---

#### Code Review


### Functional Tests


### Final steps
